src/arguments.cpp

```diff
@@ -433,6 +434,9 @@ Error Arguments::parse(const char* args) {
             CASE("inverted")
                 _inverted = true;
 
+            CASE("record-cpu")
```

Comment:
This is a section for FlameGraph options as the comment above states.
I'd put new option next to `sched`, which is kind of similar behavior-wise.